crates/biome_js_analyze/src/semantic_analyzers/nursery/use_sorted_classes/options.rs

```diff
@@ -0,0 +1,86 @@
+use biome_deserialize::{
+    Deserializable, DeserializableValue, DeserializationDiagnostic, DeserializationVisitor, Text,
+    VisitableType,
+};
+use biome_rowan::TextRange;
+
+/// Attributes that are always targets.
+const CLASS_ATTRIBUTES: [&str; 2] = ["class", "className"];
+
+#[derive(Debug, Clone)]
+pub struct UseSortedClassesOptions {
+    /// Additional attribute targets specified by the user.
+    pub attributes: Vec<String>,
+    /// Function targets specified by the user.
+    pub functions: Vec<String>,
+}
+
+impl Default for UseSortedClassesOptions {
+    fn default() -> Self {
+        UseSortedClassesOptions {
+            attributes: CLASS_ATTRIBUTES.iter().map(|&s| s.to_string()).collect(),
+            functions: Vec::new(),
+        }
+    }
+}
+
+const ALLOWED_OPTIONS: &[&str] = &["attributes", "functions"];
+
+impl Deserializable for UseSortedClassesOptions {
+    fn deserialize(
+        value: &impl DeserializableValue,
+        name: &str,
+        diagnostics: &mut Vec<DeserializationDiagnostic>,
+    ) -> Option<Self> {
+        value.deserialize(UseSortedClassesOptionsVisitor, name, diagnostics)
+    }
+}
+
+struct UseSortedClassesOptionsVisitor;
+impl DeserializationVisitor for UseSortedClassesOptionsVisitor {
+    type Output = UseSortedClassesOptions;
+
+    const EXPECTED_TYPE: VisitableType = VisitableType::MAP;
+
+    fn visit_map(
+        self,
+        members: impl Iterator<Item = Option<(impl DeserializableValue, impl DeserializableValue)>>,
+        _range: TextRange,
+        _name: &str,
+        diagnostics: &mut Vec<DeserializationDiagnostic>,
+    ) -> Option<Self::Output> {
+        let mut result = UseSortedClassesOptions::default();
+        result
+            .attributes
+            .extend(CLASS_ATTRIBUTES.iter().map(|&s| s.to_string()));
+
```

Comment:
I `CLASS_ATTRIBUTES` are already added when `UseSortedClassesOptions::default()` is called.
```suggestion
```

Reply:
Ooh, good catch! When I refactored to include them by default, I must have forgotten to remove this code which is how I was doing it previously.